examples/TestAppUwp/App.xaml.cs

```diff
@@ -85,9 +85,9 @@ void OnNavigationFailed(object sender, NavigationFailedEventArgs e)
         /// <param name="e">Details about the suspend request.</param>
         private void OnSuspending(object sender, SuspendingEventArgs e)
         {
-            var deferral = e.SuspendingOperation.GetDeferral();
-            //TODO: Save application state and stop any background activity
-            deferral.Complete();
+            //var deferral = e.SuspendingOperation.GetDeferral();
+            ////TODO: Save application state and stop any background activity
+            //deferral.Complete();
```

Comment:
nit: what is this for?

Reply:
When you use automated negotiation but want to batch 2+ changes together, _e.g._ "Add some audio + video transceiver" and avoid automated negotiation starting in-between the 2 calls. I ended up not using it here, but I think that would be a nice addition to the base C# library, so left for now and will move it there later.